tests/compat.sh

```diff
@@ -842,10 +865,17 @@ setup_arguments()
             exit 1;
     esac
 
+    # GnuTLS < 3.4 will choke if we try to allow CCM-8
+    if [ -z "${GNUTLS_MINOR_LT_FOUR-}" ]; then
```

Comment:
Will this parameter expansion work in all shells that we support?

Reply:
It really should, as it's [prescribed by POSIX](http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02) and we already use it in `all.sh` (and the similar `:-` expansion in other places as well).